src/contexts/context-utils.ts

```diff
@@ -1,14 +1,26 @@
 import { startCase } from "lodash";
 import { IconStyle, StyleResponse } from "./OntologyStyles";
 
-const hasFragment = (uri: string) =>
-  uri && uri.startsWith("http") && uri.includes("#");
+const PREFIX_LOOKUP: Record<string, string> = {
+  "http://telicent.io/ontology/": "telicent:",
+};
+
+const hasFragment = (uri: string) => uri && uri.startsWith("http") && uri.includes("#");
 
-export const getURIFragment = (uri: string) => {
+export const getOntologyClass = (uri: string) => {
   if (hasFragment(uri)) {
     const uriParts = uri.split("#");
     return uriParts.length > 1 ? uriParts[1] : uri;
   }
+
+  const foundNamespace = Object.keys(PREFIX_LOOKUP).find((namespace) => {
+    return uri.startsWith(namespace);
+  });
+
+  if (foundNamespace) {
+    return uri.replace(foundNamespace, "");
+  }
+
```

Comment:
How come you don't use the `ontologyservice.shorten`?

It has an object lookup with known uri's in, if yours isn't there you can add it usingi `ontologyservice.addPrefix`

Reply:
I didn't use it because you said the rdf services does not store the prefixes. I left a comment in the ticket so the convo can be tracked. This was just a temp solution. From my understanding addPrefix does not store/save the prefix. Also I don't always know what the prefix should be. This function isn't interested in the prefix more in finding the label which the data does not provide. I'm hoping with the changes to styles endpoint, this can provided making this function obsolete.

Reply:
If this is temporary code that'll be deleted, then I don't actually mind. 

We do need to track all of these icon work-arounds tho, so [I've added this PR here](https://telicent.atlassian.net/browse/TELFE-42)

Reply:
Sorry what I meant was it currently does not persist the lookup table, so if you refresh it you will need to add any prefixes that aren't there by default.

Does this one store them in local storage somewhere? I can't see anywhere in this PR that leads me to believe that, it looks like it just adds it to your `PREFIX_LOOKUP` object.

But there is a `getPrefix` method  on the ontologyservice already that you send the long uri and if it matches it brings back the prefix alias back plus the identier. e.g. `http://telicent.io/Person` would return `telicent:Person`

 This may be more robust because it doesn't just split fragments based on the hash, which not all uri's have. I may have the wrong end of the stick though.